go.mod

```diff
@@ -4,7 +4,6 @@ go 1.13
 
 require (
 	cloud.google.com/go v0.47.0 // indirect
-	cloud.google.com/go/storage v1.0.0
```

Comment:
I'm not sure why this change?

Reply:
Me neither.. `go mod` wonders… it comes, it goes those two things.. I have no idea why…